include/deal.II/base/types.h

```diff
@@ -41,6 +41,12 @@ namespace types
    */
   typedef unsigned int subdomain_id;
 
+  /**
+   * The type used for global indices of vertices.
+   */
+  typedef unsigned long long int global_vertex_index;
+#  define DEAL_II_VERTEX_INDEX_MPI_TYPE MPI_UNSIGNED_LONG_LONG
```

Comment:
> So no need to do something different in 32 bit mode.
> but a separate typedef would still be nice :-)

Alright, I see myself outvoted. I surrender... ;-)
I agree that there is documentation value in such a typedef, so I shall be fine with this variant.